pymc/distributions/continuous.py

```diff
@@ -336,6 +336,11 @@ def logcdf(value, lower, upper):
             ),
         )
 
+    def get_moment(value, size, lower, upper):
+        lower = at.full(size, lower, dtype=value.owner.op.inputs[2])
```

Comment:
```suggestion
        lower = at.full(size, lower, dtype=aesara.config.floatX)
```

Reply:
Because if the user passes a dtype to the distribution, using floatX might ignore it.

Reply:
And we don’t want that. If the user doesn’t specify anything, the ops dtype will already be floatX